src/Rarity/utils.ts

```diff
@@ -1,13 +1,8 @@
-import { env } from 'decentraland-commons'
 import { ContractName, getContract } from 'decentraland-transactions'
 import { ethers } from 'ethers'
 import { getMappedChainIdForCurrentChainName } from '../ethereum/utils'
 import { getRpcUrl } from '../utils/eth'
 
-export function isUsingRaritiesWithOracle(): boolean {
-  return env.get<string | undefined>('FF_RARITIES_WITH_ORACLE') === '1'
```

Comment:
Would you mind removing the `FF_RARITIES_WITH_ORACLE` env variables from the CI and from the example?